extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -414,114 +409,112 @@ private function getBadgeMsgKeyPart($badgeTypeId) {
-			$msgKeyPart =  'community-platinum';
+		if( $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
+			$msgKeyPart = 'edit-plus-category';
+		elseif( $badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM )
+			$msgKeyPart = 'community-platinum';
 		else
-			$msgKeyPart = $this->getBadgeMsgKeyPart($badgeTypeId);
+			$msgKeyPart = $this->getBadgeMsgKeyPart( $badgeTypeId );
 
 		return "achievements-badge-to-get-{$msgKeyPart}-details";
 	}
 
-	public function getBadgePictureName($badgeTypeId, $badge_lap = null, $postfix = null, $withPrefix = true) {
-		if($badgeTypeId == BADGE_LUCKYEDIT) {
+	public function getBadgePictureName( $badgeTypeId, $badge_lap = null, $postfix = null, $withPrefix = true ) {
+		if( $badgeTypeId == BADGE_LUCKYEDIT ) {
 			$badge_lap = null;
 		}
-		return (($withPrefix) ? ACHIEVEMENTS_BADGE_PREFIX : null) . $this->getBadgeMsgKeyPart($badgeTypeId). (($badge_lap !== null) ? "-{$badge_lap}" : null) . (($postfix !== null) ? "-{$postfix}" : null) . '.png';
+		return ( ( $withPrefix ) ? ACHIEVEMENTS_BADGE_PREFIX : null ) . $this->getBadgeMsgKeyPart( $badgeTypeId ) . ( ( $badge_lap !== null ) ? "-{$badge_lap}" : null ) . ( ( $postfix !== null ) ? "-{$postfix}" : null ) . '.png';
 	}
 
 	public function getHoverPictureName( $badgeTypeId ) {
 		return ACHIEVEMENTS_HOVER_PREFIX . $this->getBadgeMsgKeyPart( $badgeTypeId ) . '.png';
 	}
 
-	public function getRequiredEvents($badgeTypeId, $badge_lap = null) {
-		if($badgeTypeId == BADGE_LUCKYEDIT) {
+	public function getRequiredEvents( $badgeTypeId, $badge_lap = null ) {
+		if( $badgeTypeId == BADGE_LUCKYEDIT ) {
 			$events = 1000 * ( $badge_lap + 1 );
-		}
-		elseif($badgeTypeId == BADGE_CAFFEINATED) {
+		} elseif( $badgeTypeId == BADGE_CAFFEINATED ) {
 			$events = 100;
-		}
-		elseif($badge_lap !== null) {
+		} elseif( $badge_lap !== null ) {
 
-			if($this->getBadgeType($badgeTypeId) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY) {
+			if( $this->getBadgeType( $badgeTypeId ) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY ) {
 				$badgeTypeId = BADGE_EDIT;
 			}
 
-			$badge_lapsCount = count($this->mInTrackStatic[$badgeTypeId]['laps']);
+			$badge_lapsCount = count( $this->mInTrackStatic[ $badgeTypeId ][ 'laps' ] );
 
-			if($badge_lap >= $badge_lapsCount) {
-				$max = $this->mInTrackStatic[$badgeTypeId]['laps'][$badge_lapsCount-1]['events'];
-				$events = ($badge_lap - $badge_lapsCount + 2) * $max;
+			if( $badge_lap >= $badge_lapsCount ) {
+				$max = $this->mInTrackStatic[ $badgeTypeId ][ 'laps' ][ $badge_lapsCount - 1 ][ 'events' ];
+				$events = ( $badge_lap - $badge_lapsCount + 2 ) * $max;
 			} else {
-				$events = $this->mInTrackStatic[$badgeTypeId]['laps'][$badge_lap]['events'];
+				$events = $this->mInTrackStatic[ $badgeTypeId ][ 'laps' ][ $badge_lap ][ 'events' ];
 			}
 		} else
 			$events = null;
 
 		return $events;
 	}
 
-	public function getTrackNameKey($badgeTypeId) {
+	public function getTrackNameKey( $badgeTypeId ) {
 		$token = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$token" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)